pkg/download/README.md

```diff
@@ -0,0 +1,22 @@
+# Download configuration
+
+This feature allows you to download the configuration from a Dynatrace tenant as Monaco files. You can use this feature to avoid starting from scratch when using Monaco.
```

Comment:
Please use the term `environment` instead of `tenant`. Environment is the term, which is used everywhere else.

Reply:
fix wording for all environment mentions